set.mm

```diff
@@ -147154,6 +147148,405 @@ of an open range of nonnegative integers (of length equal to the length of
   $}
 
 
+$(
+-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-
+  Prefixes of a word
+-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-
+
+  A prefix is an arbitrary substring/subword starting at the
+  beginning of a string/word (see Wikipedia "Substring"
+  ~ https://en.wikipedia.org/wiki/Substring#Prefix or
+  ~ https://math.stackexchange.com/questions/2190559/
+  is-there-standard-terminology-notation-for-the-prefix-of-a-word ).
+
+  This is the meaning of "prefix" in computer science (automata theory or
+  formal languages) , whereas in linguistics, a prefix has a meaning (see
+  Wikipedia "Prefix" ~ https://en.wikipedia.org/wiki/Prefix ).
```

Comment:
If you want. But that's the same with every word: "ball", "union"... do not have the same meaning in everyday language.

Reply:
I added it (and would like to keep it) mainly because the Wikipedia page "prefix" is about the linguistic meaning...

Reply:
Ok. Then, "has a DIFFERENT meaning"

Reply:
The whole section header comment is removed now.